From: Jason Rhinelander <jason@imaginary.ca>
Date: Thu, 15 Dec 2022 11:52:01 -0400
Subject: Move tomli to setup_requires

It is only required for building, not for using, and breaks the package
on bullseye and earlier (where it isn't available).
---
 setup.cfg | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/setup.cfg b/setup.cfg
index eae51f0..26bf39d 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -21,5 +21,6 @@ install_requires=
     oxenmq
     oxenc
     pyonionreq
-    tomli
     sqlalchemy
+setup_requires=
+    tomli
